src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,4 +1,13 @@
-// Add the required types and props
-export const UserInfo = () => (
-  <>UserInfo markup</>
+import React from 'react';
+import { User } from '../../types/User';
+
+interface Props {
+  user: User | null;
```

Comment:
```suggestion
  user: User;
```
check the user in the `TodoInfo` component instead and don't render `UserInfo` component at all if the user doesn't exist